Packages.props

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This looks like a mistaken checkin? Doesn't appear to be imported by anything, and C# projects should handle it themselves.  

Instead, we should have a root folder Directory.Packages.Props, with content like this:

```
<Project>
  <ItemGroup>
    <PackageVersion Include="Microsoft.Windows.CSWinRT" Version="$(CppWinRTVersion)"/>
    etc...
  </ItemGroup>
</Project>
```

The root Directory.Build.Props ensures that eng\Versions.props defines the version properties above.